src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRunner.cs

```diff
@@ -100,9 +100,20 @@ private static async Task<IReadOnlyList<RestoreSummary>> RunAsync(
         /// <summary>
         /// Execute and commit restore requests.
         /// </summary>
-        public static async Task<IReadOnlyList<RestoreResultPair>> RunWithoutCommit(
+        public static Task<IReadOnlyList<RestoreResultPair>> RunWithoutCommit(
             IEnumerable<RestoreSummaryRequest> restoreRequests,
             RestoreArgs restoreContext)
+        {
+            return RunWithoutCommitAsync(restoreRequests, restoreContext, CancellationToken.None);
+        }
+
+        /// <summary>
+        /// Execute and commit restore requests.
+        /// </summary>
+        public static async Task<IReadOnlyList<RestoreResultPair>> RunWithoutCommitAsync(
+            IEnumerable<RestoreSummaryRequest> restoreRequests,
+            RestoreArgs restoreContext,
+            CancellationToken cancellationToken)
```

Comment:
This is not being used anywhere. 
You might've had a use at some point, but for now, maybe just leave it out?

Reply:
What's not being used anywhere?

The cancellation token is used around line 141, and this RunWithoutCommitAsync method is called around line 68.